Backpack/Theme/Classes/BPKSilverTheme.m

```diff
@@ -157,4 +157,12 @@ - (Class)themeContainerClass {
     return [BPKSilverThemeContainer class];
 }
 
+- (UIColor *)starColor {
+    return BPKColor.gray100;
+}
+
+- (UIColor *)starFilledColor {
+    return BPKColor.yellow500;
```

Comment:
```suggestion
    return BPKColor.red700;
```

Our "Silver" theme is used to ensure that everything that can change will, so can we make this some outrageous colour (eg red)

Reply:
Done